.github/workflows/ci.yml

```diff
@@ -29,13 +29,8 @@ jobs:
         with:
           java-version: ${{ matrix.jdk }}
 
-      - name: Check formatting
-        run: sbt "++${{ matrix.scala-version }} scalafmtCheckAll" scalafmtSbtCheck
-
-        # Consider joining this action with the "Check formatting" one once Scalafix is supported for Scala 3.
-      - name: Run linter
-        if: startsWith(matrix.scala, '2')
-        run: sbt "++${{ matrix.scala-version }} scalafixCheckAll"
+      - name: Check formatting and Scalafix
+        run: sbt "++${{ matrix.scala-version }}" scalafmtCheckAll scalafmtSbtCheck scalafixCheckAll
```

Comment:
The difference in quotes is subtle, but significant. Here we actually want both `scalafmtCheckAll` and `scalafixCheckAll` in the same command as the version change - see my explanation about the differences between them in #1188. Can you please change this to either:

- `sbt "++${{ matrix.scala-version }} scalafmtCheckAll scalafixCheckAll" scalafmtSbtCheck`
- `sbt "++${{ matrix.scala-version }} scalafmtCheckAll" "++${{ matrix.scala-version }} scalafixCheckAll" scalafmtSbtCheck`

(I don't know if the first version works as is, so please test before submitting)

Reply:
The second one works!